params/types/genesisT/genesis.go

```diff
@@ -32,6 +32,9 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
+//go:generate go run github.com/fjl/gencodec -type Genesis -field-override genesisSpecMarshaling -out gen_genesis.go
+//go:generate go run github.com/fjl/gencodec -type GenesisAccount -field-override genesisAccountMarshaling -out gen_genesis_account.go
```

Comment:
What if we had a CI test that made sure developers don't forget to run this. Could just check no-change after run?

Reply:
Great point @meowsbits